Source/ACE.Entity/Enum/MotionCommand.cs

```diff
@@ -6,415 +6,415 @@
-        MagicPowerUp02Purple,
-        MagicPowerUp03Purple,
-        MagicPowerUp04Purple,
-        MagicPowerUp05Purple,
-        MagicPowerUp06Purple,
-        MagicPowerUp07Purple,
-        MagicPowerUp08Purple,
-        MagicPowerUp09Purple,
-        MagicPowerUp10Purple,
-        Helper,
-        Pickup5,
-        Pickup10,
-        Pickup15,
-        Pickup20,
-        HouseRecall,
-        AtlatlCombat,
-        ThrownShieldCombat,
-        SitState,
-        SitCrossleggedState,
-        SitBackState,
-        PointLeftState,
-        PointRightState,
-        TalktotheHandState,
-        PointDownState,
-        DrudgeDanceState,
-        PossumState,
-        ReadState,
-        ThinkerState,
-        HaveASeatState,
-        AtEaseState,
-        NudgeLeft,
-        NudgeRight,
-        PointLeft,
-        PointRight,
-        PointDown,
-        Knock,
-        ScanHorizon,
-        DrudgeDance,
-        HaveASeat,
-        LifestoneRecall,
-        CharacterOptionsPanel,
-        SoundAndGraphicsPanel,
-        HelpfulSpellsPanel,
-        HarmfulSpellsPanel,
-        CharacterInformationPanel,
-        LinkStatusPanel,
-        VitaePanel,
-        ShareFellowshipXP,
-        ShareFellowshipLoot,
-        AcceptCorpseLooting,
-        IgnoreTradeRequests,
-        DisableWeather,
-        DisableHouseEffect,
-        SideBySideVitals,
-        ShowRadarCoordinates,
-        ShowSpellDurations,
-        MuteOnLosingFocus,
-        Fishing,
-        MarketplaceRecall,
-        EnterPKLite,
-        AllegianceChat,
-        AutomaticallyAcceptFellowshipRequests,
-        Reply,
-        MonarchReply,
-        PatronReply,
-        ToggleCraftingChanceOfSuccessDialog,
-        UseClosestUnopenedCorpse,
-        UseNextUnopenedCorpse,
-        IssueSlashCommand,
-        AllegianceHometownRecall,
-        PKArenaRecall,
-        OffhandSlashHigh,
-        OffhandSlashMed,
-        OffhandSlashLow,
-        OffhandThrustHigh,
-        OffhandThrustMed,
-        OffhandThrustLow,
-        OffhandDoubleSlashLow,
-        OffhandDoubleSlashMed,
-        OffhandDoubleSlashHigh,
-        OffhandTripleSlashLow,
-        OffhandTripleSlashMed,
-        OffhandTripleSlashHigh,
-        OffhandDoubleThrustLow,
-        OffhandDoubleThrustMed,
-        OffhandDoubleThrustHigh,
-        OffhandTripleThrustLow,
-        OffhandTripleThrustMed,
-        OffhandTripleThrustHigh,
-        OffhandKick,
-        AttackHigh4,
-        AttackMed4,
-        AttackLow4,
-        AttackHigh5,
-        AttackMed5,
-        AttackLow5,
-        AttackHigh6,
-        AttackMed6,
-        AttackLow6,
-        PunchFastHigh,
-        PunchFastMed,
-        PunchFastLow,
-        PunchSlowHigh,
-        PunchSlowMed,
-        PunchSlowLow,
-        OffhandPunchFastHigh,
-        OffhandPunchFastMed,
-        OffhandPunchFastLow,
-        OffhandPunchSlowHigh,
-        OffhandPunchSlowMed,
-        OffhandPunchSlowLow
+        Invalid = 0x0,
```

Comment:
Could you use Og's enum value alignment plugin, so we don't get another PR where we have an entire file rewritten over alignment?